lib/client-js/rooms.js

```diff
@@ -106,23 +112,30 @@ function deactivateAllRooms() {
     var roomClass = 'room-group';
     var room = document.getElementsByClassName(roomClass);
 
-    for (var i = 0; i < room.length; i++) { // Remove active-room from all rooms
-        room[i].classList.remove('active-room');
-    }
+    // for (var i = 0; i < room.length; i++) { // Remove active-room from all rooms
+    //     room[i].classList.remove('active-room');
+    // }
+    //refactored for loop to for of loop
+    for (const r of room) {
+        r.classList.remove('active-room');
+      }
     debug.end();
 }
 
 // Swap which floor of a building is being displayed
 function changeFloor(building, floor) {
+   
```

Comment:
Generally you should leave no more than 1 newline. Of course, we don't currently have a linter setup to enforce this.